deployments/types-millau.json

```diff
@@ -0,0 +1,100 @@
+{
+	"Balance": "u64",
+	"Address": "AccountId",
+	"LookupSource": "AccountId",
+	"MillauBlockHash": "H512",
```

Comment:
As mentioned earlier already, I'd prefer to split chain-related definitions from actual overwrites. I.e.
```json
"__AllRialtoThings:": "",
"RialtoBlockHash": "H256",
"RialtoBalance": "u64",
"...": "...",
"__AllMillauThings": "",
"MillauBlockHash": "H256",
"MillauBalance": "u128",
"__ChainOverrides": "",
"Balance": "MillauBalance",
"Address": "AccountId",
...
```


Do you mind grouping things like that?

Since the top part (Rialto & Millau) should be identical for both files, the next logical step is to move it into a separate `JSON` file and add a small tool that generates `types-millau.json` and `types-rialto.json` by merging `types-common.json + types-millau.json` and `types-common.json + types-rialto.json` accordingly (that should be a separate PR though).

Reply:
of course, i will set the chain related definitions. Building the tool was not so easy from the JS point of view because of async loading file process ( i tried already and it will take more time than the one i was ready to spend )

Reply:
You can use `jq` tool and create a bash script instead.
```
jq -s '.[0] * .[1]' file1,json file2.json
```